js/cljdoc.ts

```diff
@@ -134,6 +134,20 @@ function toggleMetaDialog() {
   }
 }
 
+function toggleArticlesTip() {
+  const tipToggler = document.getElementById("js--articles-tip-toggler");
+  const tip = document.getElementById("js--articles-tip");
+  if (tipToggler && tip) {
+    tipToggler.onclick = () => {
+      if (tip.classList.contains("dn")) {
+        tip.classList.remove("dn");
+      } else {
+        tip.classList.add("dn");
+      }
```

Comment:
This could be replaced with `tip.classList.toggle("dn")` and it would be a bit clearer.

Reply:
Oh thank you! will do!